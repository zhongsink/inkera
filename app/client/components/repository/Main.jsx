import React from 'react';
import List from './List';
import Languages from './Languages';
import './styles/Main.less';

const Main = () => (
  <div className="repositories-main">
    <List />
    <Languages />
  </div>
);

export default Main;
