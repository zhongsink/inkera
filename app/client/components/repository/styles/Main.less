.repositories-main {
  margin: 0 auto;
  display: flex;
}
