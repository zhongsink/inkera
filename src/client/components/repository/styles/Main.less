.repositories-main {
  width: 60%;
  margin: 0 auto;
  display: flex;
}
