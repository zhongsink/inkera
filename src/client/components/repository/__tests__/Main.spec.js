import Main from '../Main';
import { snapshot } from '../../../../shared/test';

describe('components/repository/Main', () => {
  snapshot({
    component: Main,
    name: 'Main',
  });
});
