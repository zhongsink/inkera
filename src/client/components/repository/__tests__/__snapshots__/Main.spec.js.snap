// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/repository/Main Main should render correctly 1`] = `
<div
  className="repositories-main"
>
  <div
    className="list"
  />
  <div
    className="languages"
  >
    <dl>
      <dt>
        Languages
      </dt>
      <dd>
        <span
          className="language"
        >
          Javascript
        </span>
        <span
          className="count"
        >
          37524
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          CSS
        </span>
        <span
          className="count"
        >
          2021
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          Typescript
        </span>
        <span
          className="count"
        >
          1838
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          HTML
        </span>
        <span
          className="count"
        >
          1192
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          Vue
        </span>
        <span
          className="count"
        >
          792
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          Ruby
        </span>
        <span
          className="count"
        >
          487
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          PHP
        </span>
        <span
          className="count"
        >
          207
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          C#
        </span>
        <span
          className="count"
        >
          133
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          CoffeeScript
        </span>
        <span
          className="count"
        >
          133
        </span>
      </dd>
      <dd>
        <span
          className="language"
        >
          Python
        </span>
        <span
          className="count"
        >
          124
        </span>
      </dd>
    </dl>
  </div>
</div>
`;
